src/agent/Cargo.toml

```diff
@@ -22,11 +22,11 @@ scopeguard = "1.0.0"
 regex = "1"
 
 async-trait = "0.1.42"
-tokio = { version = "0.2", features = ["rt-core", "sync", "uds", "stream", "macros", "io-util", "time", "signal", "io-std", "process", "blocking"] }
-futures = "0.3"
-netlink-sys = { version = "0.4.0", features = ["tokio_socket",]}
-tokio-vsock = "0.2.2"
-rtnetlink = "0.6.0"
+tokio = { version = "1.0.1", features = ["rt", "sync", "macros", "io-util", "time", "signal", "io-std", "process"] }
```

Comment:
Tokio is at 1.1.0 now.

Reply:
OK, will do that before ready to review.

Reply:
Any update on this @Tim-Zhang? The shutdown + tracing work (#411) needs the latest tokio code so this is a blocker for tracing.

/cc @liubin.

Reply:
@jodh-intel The pr is ready to review now, PTAL, Thanks

Reply:
Ping @Tim-Zhang, or @lifupan, @liubin :smile: